firebase.json

```diff
@@ -0,0 +1,43 @@
+{
+  "hosting": {
+    "public": "dist",
+    "rewrites": [
+      {
+        "source": "/ping",
+        "function": "ping"
+      },
+      {
+        "source": "**",
```

Comment:
this is a bit confusing because it's in the elv-live but this function is for elv-media-wallet